src/lib/openapi/spec/environment-project-schema.ts

```diff
@@ -5,29 +5,51 @@ export const environmentProjectSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['name', 'type', 'enabled'],
+    description:
+        "Describes an environment as it's configured for the project it's connected to",
```

Comment:
Non-blocking and up to you, but I feel like this is actually describing a project and not an environment. Maybe we can flip it on its head? It's a tricky thing to explain, this 😅 

```suggestion
        "Describes a project's configuration in a given environment.",
```